### DISCUSSION TITLE
stoRy 0.2.0

### DISCUSSION THREAD

**Discussion by paul-sheridan** [theme-ontology/stoRy#3] [Announcements]
This is a complete redesign of the package from the ground up.

- LTO data can now be downloaded, cached, and analyzed.
- Made package Tidyverse friendly.
- Integrated The Twilight Zone LTO v0.3.3 demo dataset.
- Removed the Star Trek episode demo data.
- Added `get_story_cluster()` function to cluster stories according to thematic similarity.
- Extensively updated documentation.

<hr /><em>This discussion was created from the release <a href='https://github.com/theme-ontology/stoRy/releases/tag/v0.2.0'>stoRy 0.2.0</a>.</em>

**Comment by odinlake** [comment]
“
Things fall apart; the centre cannot hold;
Mere anarchy is loosed upon the world,
The blood-dimmed tide is loosed, and everywhere
The ceremony of innocence is drowned;
“
